components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -170,6 +186,49 @@ private ImageAnnotationLink annotateImage(Image image, Annotation annotation) th
 
         return annotationObjects;
     }
+    
+    /* create two tag sets */
```

Comment:
if breaking these out into convenience methods, maybe make the number of annotations to be created a method argument? then makes it easier from calling side to see what it does.